docs/library/zephyr.FlashArea.rst

```diff
@@ -0,0 +1,29 @@
+.. currentmodule:: zephyr
+.. _zephyr.FlashArea:
+
+class FlashArea -- access to built-in flash storage
+===================================================
+
+Uses zephyr flash map API.
```

Comment:
as above, would be good to add some info about how flash map differs from disk access